docs/spec/spec.md

```diff
@@ -185,8 +185,8 @@ spec:
         livenessProbe: ...  # Optional
         readinessProbe: ...  # Optional
 
-      # +optional concurrency strategy.  Defaults to Multi.
-      concurrencyModel: ...
```

Comment:
Are we removing the existing `concurrencyModel`? That will break any existing users who have it set.

I'd rather see this remain documented as a deprecated legacy mechanism.